src/knock.ts

```diff
@@ -86,28 +86,29 @@ class Knock {
     // Default to 1 hour from now
     const expireInSeconds = options.expiresInSeconds ?? 60 * 60;
 
-    return jwt.sign(
-      {
-        sub: userId,
-        iat: currentTime,
-        exp: currentTime + expireInSeconds,
-      },
-      signingKey,
-      {
-        algorithm: "RS256",
-      },
-    );
+    return await new SignJWT({
+      sub: userId,
+      iat: currentTime,
+      exp: currentTime + expireInSeconds,
+    })
+      .setProtectedHeader({ alg: "RS256", typ: "JWT" })
+      .sign(new TextEncoder().encode(signingKey));
```

Comment:
I pulled down the example Next app to play around with, and get this error when trying to generate a signing key. Entirely possible I'm doing something wrong though: <img width="1085" alt="Screenshot 2024-01-05 at 1 14 49 PM" src="https://github.com/knocklabs/knock-node/assets/7818951/02728b89-32fe-46d9-9dfa-e3ba70c5ba52">

Reply:
@JEverhart383 Jeff do you think this is something you could take another look/check to see if you are still getting the error above?

Reply:
Tested and this is fixed